builder/dockerfile/copy.go

```diff
@@ -56,6 +56,7 @@ type copyInstruction struct {
 	cmdName                 string
 	infos                   []copyInfo
 	dest                    string
+	flChown                 string
```

Comment:
@dnephin it's a string serialized `uid:gid` id pair, maybe it worth to parse the chown option earluer & split into uid & gid in the struct in place of a single string?